pkg/types/types.go

```diff
@@ -4,12 +4,90 @@
 package types
 
 import (
+	"path"
+	"strings"
+
+	"github.com/square/p2/pkg/util"
+
+	"github.com/pborman/uuid"
```

Comment:
missed this question, but yeah it's what we use to generate uuids for RCs and pod clusters